### PR TITLE
feat/Footer

### PR DESCRIPTION
**Pull request by Quannichan**
Feat: design footer layout, Refactor: add portfolio and fix some main component

### CODE REVIEW

File: src/stylesheet/components/plan_tags.scss [hunk 1]
```diff
@@ -259,4 +259,4 @@
     #wh_scr{
         color: $color_white;
     }
-}
+}
```
Comment (Lanng21) [inline src/stylesheet/components/plan_tags.scss]: end of line

File: src/stylesheet/components/testimonials_tag.scss [hunk 2]
```diff
@@ -191,4 +191,4 @@
         justify-content: left;
         column-gap: 20px;
     }
-}
+}
```
Comment (Lanng21) [inline src/stylesheet/components/testimonials_tag.scss]: same

File: src/stylesheet/components/stay_intouch_form.scss [hunk 3]
```diff
@@ -86,4 +86,4 @@
         justify-content: center;
         align-items: center;
     }
-}
+}
```
Comment (Lanng21) [inline src/stylesheet/components/stay_intouch_form.scss]: same

File: src/stylesheet/layouts/large_screen/_footer_large.scss [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+    .wrap_trademark_footer{
+        padding: 1%;
+    }
+    .trademark_footer{
+        @include trademark_large();
+        color: $color_yellow;
+    }
+    .wrap_footer_script{
+        text-align: left;
+        width: 370px
```
Comment (Lanng21) [inline src/stylesheet/layouts/large_screen/_footer_large.scss]: dont fix width. u should use max-width and use % for it

File: src/stylesheet/layouts/large_screen/_main_large.scss [hunk 5]
```diff
@@ -262,4 +262,174 @@
...
+    .header_img{
+        @include header_font_medium();
+        margin-bottom: 10px
+    }
+    .time_img{
+        @include font_large();
+        margin-bottom: 20px;
+    }
+    .scr_img{
+        width: 230px;
```
Comment (Lanng21) [inline src/stylesheet/layouts/large_screen/_main_large.scss]: dont set w h for img

Comment (Quannichan) [reply src/stylesheet/layouts/large_screen/_main_large.scss]: This is scriprt of image and it is p tag, not an image 🤧

Comment (Lanng21) [reply src/stylesheet/layouts/large_screen/_main_large.scss]: same

**Review by Lanng21 — APPROVED** [review]

**Pull request merged**